aptos-move/framework/src/built_package.rs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::error_map::generate_error_map;
+use crate::natives::code::{ModuleMetadata, PackageMetadata, UpgradePolicy};
+use aptos_types::account_address::AccountAddress;
+use clap::Parser;
+use move_deps::move_core_types::errmap::ErrorMapping;
+use move_deps::move_package::compilation::compiled_package::CompiledPackage;
+use move_deps::move_package::BuildConfig;
+use serde::{Deserialize, Serialize};
+use std::collections::BTreeMap;
+use std::path::PathBuf;
+
+/// Represents a set of options for building artifacts from Move.
+#[derive(Debug, Clone, Parser, Serialize, Deserialize)]
+pub struct BuildOptions {
```

Comment:
after this change, would the mv files auto-generated once the source move files change?

Reply:
The release bundle will *not* be autogenerated because this leads to a fragile build environment. Rather there is a test that everything is up-to-date. If that test fails, you have to regenerate manually.